examples/embeddings_extraction.py

```diff
@@ -0,0 +1,48 @@
+import torch
+
+from farm.data_handler.processor import GNADProcessor
+from farm.infer import Inferencer
+from farm.modeling.adaptive_model import AdaptiveModel
+from farm.modeling.language_model import Bert
+from farm.modeling.tokenization import BertTokenizer
+
+from farm.utils import set_all_seeds, MLFlowLogger, initialize_device_settings
+
+##########################
+########## Settings
+##########################
+set_all_seeds(seed=42)
+device, n_gpu = initialize_device_settings(use_cuda=True)
+batch_size = 32
+lang_model = "bert-base-german-cased"
+
+# 1.Create a tokenizer
+tokenizer = BertTokenizer.from_pretrained(
+    pretrained_model_name_or_path=lang_model, do_lower_case=False
+)
+
+# 2. Create a DataProcessor that handles all the conversion from raw text into a pytorch Dataset
+processor = GNADProcessor(
```

Comment:
This is fine for now, but we will need to switch this processor here to a more generic one. Something like a "BasicProcessor". Since we already had something similar in mind to simplify most of the text classification processors, I wouldn't do it right now here, but in a separate MR later.